article/_posts/2020-09-10-Munchausen_Reinforcement_Learning.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: review
+title: "Munchausen Reinforcement Learning"
+tags: reinforcement
+author: "Antoine Théberge"
+cite:
+    authors: "Nino Vieillard, Olivier Pietquin, Matthieu Geist"
+    title:   "Munchausen Reinforcement Learning"
+    venue:   "https://arxiv.org/abs/2007.14430"
+pdf: ""
+---
+
+
+# Highlights
+
+- Novel term added to DQN's target, akin to maximum-entropy reinforcement learning, achieving state-of-the-art results on the Atari Learning Environment suite
+
+
+![](/article/images/MRL/munchausen.jpg)
+> Baron Munchausen pulls himself out of a mire by his own hair.
+
+> source: [https://en.wikipedia.org/wiki/M%C3%BCnchhausen_trilemma](https://en.wikipedia.org/wiki/M%C3%BCnchhausen_trilemma)
+
+# Introduction
+
+Reinforcement learning (RL) often uses _bootstrapping_ when training agents: bootstrapping occurs when the target for computing a loss function leverages the network's own estimates, like with Deep Q-Network (DQN)'s Temporal-Difference (TD) update: 
+
+$$\tag{1} L(\phi) = \mathbb{E}_{s,a,s',r \sim D}\big(\underbrace{r + \gamma \max_{a'}Q_{\phi'}(s', a')}_{\text{target}} - Q_\phi(s,a)\big)^2$$
+
+The authors propose to leverage another estimate from the agent: it's own policy. It is similar to maximum-entropy RL, which attempts to maximize both the expected return and the policy's entropy s.t. the RL problem becomes:
```

Comment:
Not sure if the last sentence is missing something: I feel that the s.t. is missing its "argument", something like
_It is similar to maximum-entropy RL, which attempts to maximize both the expected return and the policy's entropy s.t. {missing}, the RL problem becoming:_

Reply:
I reformulated it in b0c97df